common/religion/religions/light_group.txt

```diff
@@ -301,5 +301,52 @@
 			doctrine = tenet_monasticism
 			doctrine = tenet_order_syncretism
 		}
+		#Light/Arcane religion
+		dalaranian_religion = { #Blend between base light religion and arcane religion
```

Comment:
About the name, I think in the dev chat we decided to go with Kirin Tor.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/113423753-15a4a080-93e0-11eb-90ee-668bab9a5e83.png)

</details>

Also, the faith will need a description. Probably something about Dalaran's past and their traditions.

Reply:
The key of religion (dalaranian_religion, kirin_tor) doesn't matter. What matters is how it's localized in `Warcraft-Guardians-of-Azeroth-2\localization\english\religion\wc_religion_l_english.yml`.

Reply:
Ah my bad

Reply:
Added a description, but I'm no writer so someone better might have to redo it.